src/extension.ts

```diff
@@ -16,8 +17,10 @@ import {
   SELECT_CONNECTION_COMMAND,
 } from './common';
 
+const logger = new Logger('extension');
+
 export function activate(context: vscode.ExtensionContext) {
-  console.log(
+  logger.info(
     'Congratulations, your extension "vscode-deephaven" is now active!'
   );
```

Comment:
I think you need to create the output channel/add the logger handlers before this gets logged.

Reply:
fixed